src/styles/block/calendar.scss

```
@@ -0,0 +1,45 @@
+.calendar {
+  margin: 0 auto;
+  display: flex;
+  flex-wrap: wrap;
+  width: ($day-size * 7) + ($gap-size * 6) + ($padding-size * 2);
+  padding: $padding-size;
+  gap: $gap-size;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: $day-size;
+    height: $day-size;
+    font-size: 30px;
+    background-color: #eee;
+    border: 1px solid black;
+    cursor: pointer;
+    transition: background-color 0.5s transform 0.5s;
+
+    &:hover {
+      background-color: pink;
+      transform: translateY(-20%);
+    }
+
+    @for $i from 1 through 31 {
+      &:nth-child(#{$i})::before {
+        content: $i + "";
+      }
+    }
+  }
+
+  @for $i from 28 through 30 {
+    &--month-length_#{$i} :nth-child(n + #{$i + 1}) {
```

```suggestion
    &--month-length--#{$i} :nth-child(n + #{$i + 1}) {
```
modifiers don't work because of it